charts/wildfly-common/templates/_ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- define "wildfly-common.ingress.tpl" -}}
+kind: Ingress
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: {{ include "wildfly-common.appName" . }}
+  labels: {}
+  annotations: {}
+spec:
+  {{- if .Values.deploy.ingress.className }}
+  ingressClassName: {{ .Values.deploy.ingress.className }}
+  {{- end }}
+  {{- if .Values.deploy.ingress.tls.enabled }}
+  tls:
+    - 
+       {{- if .Values.deploy.ingress.host }}
+      hosts: 
+        - {{ .Values.deploy.ingress.host }}
+      {{- end }}
+      {{- include "wildfly-common.deployment.ingressSecret" . | nindent 6 -}}
+  {{- end }}
+  rules:
+    - 
+      {{- if .Values.deploy.ingress.host }}
+      host: {{ .Values.deploy.ingress.host }}
+      {{- end }}
+      http:
+        paths:
+          - path: /
```

Comment:
I wonder if we should also make the path configurable?
with the current setting, there can be only a single service for a given host but it is reasonable to expose multiple services on the same host. 
Having `/` as the default is fine but maybe we could add a `deploy.tls.path` attribute to change this. wdyt?

Reply:
Also currently we use the same value for TLS *hosts* and *host* while *host* can be an expression.
It depends on how far we want to make the ingress resource customizable

Reply:
I think we should stick to a simple ingress configuration (host, path, tls).
If the user needs more configuration, they can create a dedicate `ingress` resource outside of the Helm chart.